heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,25 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean consolePrompt(int oldContainerCount, int newContainerCount) {
+    String fmt =
+        "The present aurora job has %d containers. After update there will be %d containers. ";
+    Console c = System.console();
+    if (c == null) {
+      LOG.warning("No console to prompt user");
```

Comment:
suggest: "No console to prompt user. Proceeding."

Reply:
updated